zulipterminal/themes/colors_gruvbox.py

```diff
@@ -0,0 +1,140 @@
+"""
+COLOR
+-----
+Contains color definitions or functions common across gruvbox themes.
+For further details on themefiles look at the theme contribution guide.
+"""
+from enum import Enum
+from typing import Any
+
+from pygments.style import Style
+from pygments.token import (
+    Comment,
+    Error,
+    Escape,
+    Generic,
+    Keyword,
+    Literal,
+    Name,
+    Number,
+    Operator,
+    Punctuation,
+    String,
+    Text,
+    Whitespace,
+)
+
+
+# fmt: off
+# NOTE: The 24bit color codes use 256 color which can be
+# enhanced to be truly 24bit.
+# NOTE: The 256code format can be moved to h0-255 to
+# make use of the complete range instead of only 216 colors.
+
+class GruvBoxColor(Enum):
+    # color          =  16code          256code   24code
```

Comment:
Actually, this file contains all required colors (for gruvbox dark as well as gruvbox light). So, can you please suggest if it would be better to have colors in theme files only (instead of having a separate file for gruvbox colors)?